### PR TITLE
Update CHANGELOG for 0.7.0 release

### PR DESCRIPTION
**Pull request by SuperQ**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+## 0.7.0 / 2016-02-12
+
+BREAKING CHANGES:
+* global status metrics for "handlers" have been renamed.
```
Comment (juliusv) [inline CHANGELOG.md]: Consistency with everything else in this file: start bullet point capitalized.

Comment (juliusv) [inline CHANGELOG.md]: and decide on periods or not :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+* [FEATURE] New server version metric (PR #59)
+* [FEATURE] New collector for `information_schema.innodb_metrics` (PR #69)
+* [FEATURE] Read credentials from ".my.cnf" files (PR #77)
+* [FEATURE] New collector for query response time distribution (PR #79)
+* [IMPROVEMENT] Collect more metrics from `performance_schema.events_statements_summary_by_digest` (PR #58)
+* [IMPROVEMENT] Add option to filter metrics queries from the slow log (PR #60)
+* [IMPROVEMENT] Leverage lock-free SHOW SLAVE STATUS (PR #61)
+* [IMPROVEMENT] Add labels to global status "handlers" counters (PR #68)
+* [IMPROVEMENT] Update Makefile.COMMON from utils repo (PR #73)
+* [BUGFIX] Fix the return of the function and logging the error (PR #64)
```
Comment (juliusv) [inline CHANGELOG.md]: Hm, this is a good example why a) good PR/commit descriptions are important and b) sometimes it's better to invent better descriptions for the changelog.

Maybe at least something like, "Fix broken error return in the scrape function and log an error instead"?

Comment (SuperQ) [inline CHANGELOG.md]: Yea, I need to be more proactive in making sure PRs are well named.

**Comment by juliusv** [comment]
:+1:

**Comment by SuperQ** [comment]
I'll probably add one more PR to this, but it's pretty simple.

**Pull request merged**